leftwm/src/config/default.rs

```diff
@@ -199,14 +201,23 @@ impl Default for Config {
             .map(|s| (*s).to_string())
             .collect();
 
+        let scratchpad = ScratchPad {
+            name: "Alacritty".into(),
+            value: "alacritty".to_string(),
+            x: Some(Size::Pixel(860)),
+            y: Some(Size::Pixel(390)),
+            height: Some(Size::Pixel(300)),
+            width: Some(Size::Pixel(200)),
+        };
+
         Self {
             workspaces: Some(vec![]),
             tags: Some(tags),
             layouts: LAYOUTS.to_vec(),
             layout_mode: LayoutMode::Workspace,
             // TODO: add sane default for scratchpad config.
             // Currently default values are set in sane_dimension fn.
```

Comment:
Is this TODO still needed?

Reply:
Good question. I don't really know as this looks to me more like the question of what the defaults should be. I added these values because these came from the wiki and gave a case to test against.

This might create the need to open a new PR which discusses what the defaults should look like?

Reply:
> This might create the need to open a new PR which discusses what the defaults should look like?

I agree